core/crypto/secp256k1.go

```diff
@@ -12,10 +12,10 @@ import (
 	"github.com/minio/sha256-simd"
 )
 
-// Secp256k1PrivateKey is an Secp256k1 private key
+// Secp256k1PrivateKey is a Secp256k1 private key
```

Comment:
I don't think that's correct.

Reply:
This is a non-vowel word.

Reply:
The comment is a very circular definition, but using `a` is right since `Secp256k1` is a name